# Examining Racial Discrimination in the US Job Market by Robert Joseph

### Background
Racial discrimination continues to be pervasive in cultures throughout the world. Researchers examined the level of racial discrimination in the United States labor market by randomly assigning identical résumés to black-sounding or white-sounding names and observing the impact on requests for interviews from employers.

### Data
In the dataset provided, each row represents a resume. The 'race' column has two values, 'b' and 'w', indicating black-sounding and white-sounding. The column 'call' has two values, 1 and 0, indicating whether the resume received a call from employers or not.

Note that the 'b' and 'w' values in race are assigned randomly to the resumes when presented to the employer.

<div class="span5 alert alert-info">
### Exercises
You will perform a statistical analysis to establish whether race has a significant impact on the rate of callbacks for resumes.

Answer the following questions **in this notebook below and submit to your Github account**. 

   1. What test is appropriate for this problem? Does CLT apply?
   2. What are the null and alternate hypotheses?
   3. Compute margin of error, confidence interval, and p-value.
   4. Write a story describing the statistical significance in the context or the original problem.
   5. Does your analysis mean that race/name is the most important factor in callback success? Why or why not? If not, how would you amend your analysis?

You can include written notes in notebook cells using Markdown: 
   - In the control panel at the top, choose Cell > Cell Type > Markdown
   - Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet


#### Resources
+ Experiment information and data source: http://www.povertyactionlab.org/evaluation/discrimination-job-market-united-states
+ Scipy statistical methods: http://docs.scipy.org/doc/scipy/reference/stats.html 
+ Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet
</div>
****

In [3]:
import pandas as pd
import numpy as np
from scipy import stats

In [4]:
data = pd.io.stata.read_stata('data/us_job_market_discrimination.dta')

# 1. What test is appropriate for this problem? Does CLT apply?
A 2-sample hypothesis test would be appropriate for this problem,and CLT would apply due to the large sample size.

In [5]:
# number of callbacks for black-sounding names
sum(data[data.race=='b'].call)

157.0

In [6]:
# number of resumes sent for black-sounding names
data[data['race']=='b'].call.size

2435

In [7]:
black_stats = data[data['race']=='b'].call.describe()
black_stats

count    2435.000000
mean        0.064476
std         0.245649
min         0.000000
25%         0.000000
50%         0.000000
75%         0.000000
max         1.000000
Name: call, dtype: float64

In [8]:
black_mean = data[data['race']=='b'].call.mean()
black_mean

0.0644763857126236

In [28]:
black_sem = data[data['race']=='b'].call.sem()

In [10]:
# number of callbacks for white-sounding names
sum(data[data.race=='w'].call)

235.0

In [11]:
# number of resumes sent for white-sounding names
data[data['race']=='b'].call.size

2435

In [12]:
white_stats = data[data['race']=='w'].call.describe()
white_stats

count    2435.000000
mean        0.096509
std         0.295346
min         0.000000
25%         0.000000
50%         0.000000
75%         0.000000
max         1.000000
Name: call, dtype: float64

In [13]:
white_mean = data[data['race']=='w'].call.mean()
white_mean

0.09650924056768417

In [29]:
white_sem = data[data['race']=='w'].call.sem()

In [30]:
# difference between means in white-sounding and black-sounding names
data_diff = white_mean-black_mean
data_diff

0.03203285485506058

In [36]:
# Standard Error Sum
bw_se = white_sem+black_sem
bw_se

0.010963361849400492

In [40]:
# 95% CI
interval = 1.96*bw_se
print ('Population mean is within ' + str(interval - data_diff) + ' and ' + str(interval + data_diff))

Population mean is within -0.0105446656302 and 0.0535210440799


# 2. What are the null and alternate hypotheses?
The null hypothesis would be that there is no difference in the means of black-sounding and white-sounding names, while the alternate would be that there is a difference in the means of black-sounding and white-sounding names.

In [41]:
# Null Hypothesis => There is no difference between white-sounding and black-sounding names
Z_score = (data_diff-0)/bw_se
Z_score

2.9218095047015367

# 3. Compute margin of error, confidence interval, and p-value.
The margin of error is 0.021488189224824965.
CI: Population mean is within -0.0105446656302 and 0.0535210440799
p-value = 0.0018000000000000238

In [44]:
interval

0.021488189224824965

In [46]:
p_val = 1-.9982
p_val

0.0018000000000000238

# 4. Write a story describing the statistical significance in the context or the original problem.
Current data suggests that giving a child a white-sounding name would improve his/her chances of getting hired, compared to giving them a black-sounding name. Analysis of a 4870 sample size of identical resumes, evenly split between balck-sounding and white-sounding names, showed a significant difference in the number of callbacks that employers made to potential hires with white-sounding names when compared to black-sounding names. The candidates with white-sounding names received roughly 1.5 times the number of calls compared to black-sounding names, and the probability of this difference being due to just chance was calculated to be two-tenths of one percent.

# 5. Does your analysis mean that race/name is the most important factor in callback success? Why or why not? If not, how would you amend your analysis?
My analysis does not mean that race/name is the most important factor in callback success because the qualifications required for the position, and if each candidate met those qualifications, were not considered. I would amend the analysis by having resumes that exactly matched the qualifications for each position for each black-sounding and white-sounding candidate.